strings/loot_data/all_spawn_paths.txt

```diff
@@ -1475,6 +1475,7 @@
 /obj/item/weapon/gun_upgrade/mechanism/overshooter
 /obj/item/weapon/gun_upgrade/mechanism/glass_widow
 /obj/item/weapon/gun_upgrade/mechanism/weintraub
+/obj/item/weapon/gun_upgrade/mechanism/funswitch
```

Comment:
This is an automatically generated file and doesn't need to be changed for this PR.

Reply:
Should hopefully be removed from the PR.